docs/New Game.md

```diff
@@ -31,14 +31,85 @@ game to have proper pickups:
 2. Update `standard_pickup_configuration` and `ammo_pickup_configuration` in your game's starter preset.
 
 
+## Exporting games
 
-### Exporting games
+Up to now, everything is just theoretical values inside Randovania. The next step is making these modifications be
+applied to the game. 
```

Comment:
This reads a bit weird. Maybe "The next step is applying these modifications to the game"?